File: patch_app.js
const fs = require('fs');

let content = fs.readFileSync('App.tsx', 'utf-8');

content = content.replace(
  "import { DaftarHadirPage } from './components/DaftarHadirPage';",
  "import { DaftarHadirAdmin } from './components/DaftarHadirAdmin';\nimport { FormKehadiran } from './components/FormKehadiran';"
);

// We should replace `<DaftarHadirPage />` with `<DaftarHadirAdmin />`
content = content.replace(
  ") : activeTab === 'daftar-hadir' ? (\n          <DaftarHadirPage />\n        )",
  ") : activeTab === 'daftar-hadir' ? (\n          <DaftarHadirAdmin />\n        )"
);
content = content.replace(
  ") : activeTab === 'daftar-hadir' ? (\n          <DaftarHadirPage />\n        )", // maybe spacing is different, let's use regex
  "xxx"
);

fs.writeFileSync('App.tsx', content);
console.log('patched app');

File: patch_app.py
import re

with open('App.tsx', 'r') as f:
    content = f.read()

import_statement = "import { DashboardRekapBmd } from './components/DashboardRekapBmd';\n"
content = "import { DashboardRekapBmd } from './components/DashboardRekapBmd';\n" + content

pantau_kgb = """         ) : activeTab === 'pantau-kgb' ? (
          <PantauKGB />
"""
dashboard_bmd = """        ) : activeTab === 'rekap-bmd' ? (
          <DashboardRekapBmd />
"""

content = content.replace(pantau_kgb, pantau_kgb + dashboard_bmd)

with open('App.tsx', 'w') as f:
    f.write(content)
